api/src/domain/updateSidebarCards.test.ts

```diff
@@ -127,6 +127,45 @@ describe("updateRoadmapSidebarCards", () => {
   });
 
   describe("welcome card", () => {
+    const operatingPhases: OperatingPhaseId[] = [
+      "GUEST_MODE",
+      "GUEST_MODE_OWNING",
+      "NEEDS_TO_FORM",
+      "NEEDS_TO_REGISTER_FOR_TAXES",
+      "FORMED_AND_REGISTERED",
+    ];
+
+    it.each<OperatingPhaseId | undefined>(operatingPhases)(
+      "removes welcome card when the updated visibleSidebarCards do NOT have welcome card",
+      (operatingPhaseId) => {
+        const userData = generateUserData({
+          profileData: generateProfileData({
+            operatingPhase: operatingPhaseId,
+            industryId: "generic",
+          }),
+          preferences: generatePreferences({ visibleSidebarCards: [] }),
+        });
+        const updatedUserData = updateSidebarCards(userData);
+        expect(updatedUserData.preferences.visibleSidebarCards).not.toContain("welcome");
+      }
+    );
+
+    it.each<OperatingPhaseId | undefined>(["UP_AND_RUNNING", "UP_AND_RUNNING_OWNING"])(
+      "removes welcome-up-and-running card when the updated visibleSidebarCards do NOT have welcome-up-and-running card",
+      (operatingPhase) => {
+        console.log(operatingPhase);
```

Comment:
remove console log

Reply:
I thought I got all of them and I was wrong. Nice catch